editor/scene_tree_editor.h

```diff
@@ -181,6 +181,7 @@ class SceneTreeDialog : public ConfirmationDialog {
 public:
 	void popup_scenetree_dialog();
 	SceneTreeEditor *get_scene_tree() { return tree; }
+	LineEdit *get_filter() { return filter; }
```

Comment:
This could get a less ambiguous name (`get_filter()`  sounds like something that could return String), like `get_filter_line_edit()`.